charts/ndb-operator/README.md

```diff
@@ -0,0 +1,171 @@
+# Nutanix Database Service Operator for Kubernetes
+The NDB operator brings automated and simplified database administration, provisioning, and life-cycle management to Kubernetes.
+
+---
+
+## Pre-requisites
+1. NDB [installation](https://portal.nutanix.com/page/documents/details?targetId=Nutanix-Era-User-Guide-v2_4:top-era-installation-c.html).
+2. A Kubernetes cluster.
+3. Helm installed.
+
+## Installation and Running on the cluster
+Deploy the operator on the cluster:
+```sh
+helm repo add nutanix https://nutanix.github.io/helm/
+
+helm install ndb-operator nutanix/ndb-operator --version 0.0.2
+```
+## Using the Operator
+
+1. Create the secrets that are to be used by the custom resource:
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: your-ndb-secret
+type: Opaque
+stringData:
+  username: username-for-ndb-server
+  password: password-for-ndb-server
+  ca_certificate: |
+    -----BEGIN CERTIFICATE-----
+    CA CERTIFICATE (ca_certificate is optional)
+    -----END CERTIFICATE-----
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: your-db-secret
+type: Opaque
+stringData:
+  password: password-for-the-database-instance
+  ssh_public_key: SSH-PUBLIC-KEY
+
+```
+2. To create instances of custom resources (provision databases), edit the crd file with the NDB installation and database instance details and run:
+```sh
+kubectl apply -f CRD_FILE.yaml
+```
+3. To delete instances of custom resources (deprovision databases) run:
+```sh
+kubectl delete -f CRD_FILE.yaml
+```
+The CRD is described as follows:
+```yaml
+apiVersion: ndb.nutanix.com/v1alpha1
+kind: Database
+metadata:
+  # This name that will be used within the kubernetes cluster
+  name: db
+spec:
+  # NDB server specific details
+  ndb:
+    # Cluster id of the cluster where the Database has to be provisioned
+    # Can be fetched from the GET /clusters endpoint
+    clusterId: "Nutanix Cluster Id" 
+    # Credentials secret name for NDB installation
+    # data: username, password, 
+    # stringData: ca_certificate
+    credentialSecret: your-ndb-secret
+    # The NDB Server
+    server: https://[NDB IP]:8443/era/v0.9
+    # Set to true to skip SSL verification, default: false.
+    skipCertificateVerification: true
+  # Database instance specific details (that is to be provisioned)
+  databaseInstance:
+    # The database instance name on NDB
+    databaseInstanceName: "Database-Instance-Name"
+    # Names of the databases on that instance
+    databaseNames:
+      - alpha
+      - beta
+    # Credentials secret name for NDB installation
+    # data: password, ssh_public_key
+    credentialSecret: your-db-secret
+    size: 10
+    timezone: "UTC"
+    type: postgres
+```
+## Uninstalling the Chart
+To uninstall/delete the operator deployment/chart:
+```console
+helm uninstall [RELEASE_NAME]
```

Comment:
let give the exact command based on the install above

helm uninstall ndb-operator -n ndb-operator